src/redux/actions/actionHelpers.js

```diff
@@ -0,0 +1,22 @@
+import { toast } from 'react-toastify';
+
+export const fetchOptions = (method) => ({
+  method,
+  headers: {
+    Accept: '*/*',
+    'Content-Type': 'application/json',
+    authorization: localStorage.getItem('token'),
+  },
+  mode: 'cors',
+});
+
+export const flashToast = (resp) => {
+  const res = resp.status === 200 ? 'success' : 'error';
+  return toast[res](resp.message);
+};
+
+const ref = 'http://localhost:3000/api/v1/articles';
```

Comment:
are we still using the localhost in staging?I think we are using common staging url not localhost